src/store/reducers/schema/schema.ts

```diff
@@ -51,61 +51,60 @@ export const schemaApi = api.injectEndpoints({
                 }
             },
         }),
-        getSchema: builder.query<TEvDescribeSchemeResult & {partial?: boolean}, {path: string}>({
+        getSchema: builder.query<{[path: string]: TEvDescribeSchemeResult}, {path: string}>({
             queryFn: async ({path}, {signal}) => {
                 try {
                     const data = await window.api.getSchema({path}, {signal});
-                    return {data: data ?? {}};
+                    return {data: data ? {[path]: data, ...getSchemaChildren(data)} : {}};
                 } catch (error) {
                     return {error};
                 }
             },
             keepUnusedDataFor: Infinity,
-            forceRefetch: ({endpointState}) => {
-                const data = endpointState?.data;
-                if (data && typeof data === 'object' && 'partial' in data && data.partial) {
-                    return true;
-                }
-                return false;
+            serializeQueryArgs: ({queryArgs: {path}}) => {
+                const parts = path.split('/');
+                return {path: parts[0] || parts[1]};
```

Comment:
How does this serialize works? Data is cached for the tenant?

Reply:
Yes